scripts/create_market_seer.py

```
@@ -0,0 +1,72 @@
+    OMEN_DEFAULT_MARKET_FEE_PERC,
+    CollateralTokenChoice,
+)
+from prediction_market_agent_tooling.tools.utils import DatetimeUTC
+
+
+# ToDo - Adapt to Seer
+
+
+def main(
+    question: str = typer.Option(),
+    closing_time: datetime = typer.Option(),
```

Ruff (0.8.2)

Do not perform function call `typer.Option` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)